### PR TITLE
Fix broken links to readthedocs.io urls

### PR DESCRIPTION
**Pull request by hackerkid**
I skipped `tasks/2016` directory.

### CODE REVIEW

File: docs/improving-gci-experience.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ all of the mentors are reading through the feed as quickly as they can,
 while helping people.
 
 If you are new to the concept of the netiquette, please read this [helpful
-resource](http://www.albion.com/netiquette/).
+resource](https://www.albion.com/netiquette/).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

File: docs/improving-gci-experience.md [hunk 2]
```diff
@@ -330,30 +330,30 @@ courses:
 
   * [Open Source Society University](https://ossu.firebaseapp.com/).
 
-  * [The Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/).
+  * [The Hitchhiker's Guide to Python](https://docs.python-guide.org/en/latest/).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

File: docs/improving-gci-experience.md [hunk 3]
```diff
@@ -330,30 +330,30 @@ courses:
...
 
 * Contribute to other OSS projects - [a list of projects good for
 beginners](https://github.com/MunGell/awesome-for-beginners).
 
 * Find other programming resources that'll help you grow, some examples:
 
   * Brandon Rhodes's [talk "The Mighty
-  Dictionary"](http://pyvideo.org/pycon-us-2010/the-mighty-dictionary-55.html).
+  Dictionary"](https://pyvideo.org/pycon-us-2010/the-mighty-dictionary-55.html).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

File: docs/improving-gci-experience.md [hunk 4]
```diff
@@ -330,30 +330,30 @@ courses:
...
 
   * Allison Kaptur's blog posts, examples:
     * [Effective learning strategies for
-    programmers](http://akaptur.com/blog/2015/10/10/effective-learning-strategies-for-programmers/).
+    programmers](https://akaptur.com/blog/2015/10/10/effective-learning-strategies-for-programmers/).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

File: docs/improving-gci-experience.md [hunk 5]
```diff
@@ -330,30 +330,30 @@ courses:
...
 
     * [Systematic
-    debugging](http://akaptur.com/blog/2013/07/24/systematic-debugging/).
+    debugging](https://akaptur.com/blog/2013/07/24/systematic-debugging/).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

File: docs/improving-gci-experience.md [hunk 6]
```diff
@@ -330,30 +330,30 @@ courses:
...
 
   * Jessica McKellar's ["Building and Breaking a Python
-  Sandbox"](http://pyvideo.org/pycon-us-2014/building-and-breaking-a-python-sandbox.html).
+  Sandbox"](https://pyvideo.org/pycon-us-2014/building-and-breaking-a-python-sandbox.html).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

File: docs/improving-gci-experience.md [hunk 7]
```diff
@@ -330,30 +330,30 @@ courses:
...
 
   * [Cache me if you can](https://us.pycon.org/2014/schedule/presentation/173/):
   memcached, caching patterns and best practices
   ([slides](https://speakerdeck.com/pycon2014/cache-me-if-you-can-memcached-caching-patterns-and-best-practices-by-guillaume-ardaud),
-  [video](http://pyvideo.org/video/2578/cache-me-if-you-can-memcached-caching-patterns)).
+  [video](https://pyvideo.org/video/2578/cache-me-if-you-can-memcached-caching-patterns)).
```
Comment (lonerz) [inline docs/improving-gci-experience.md]: HTTPS doesn't work here.

**Comment by hackerkid** [comment]
@lonerz Fixed the issues.

**Comment by timabbott** [comment]
Merged, thanks @hackerkid!

**Pull request closed without merging**